b2/pom.xml

```diff
@@ -41,20 +41,6 @@
     <jclouds.osgi.import>org.jclouds*;version="${project.version}",*</jclouds.osgi.import>
   </properties>
 
-  <!-- For modernizer, which depends on jclouds-resources snapshot. -->
-  <pluginRepositories>
-    <pluginRepository>
-      <id>apache-snapshots</id>
-      <url>https://repository.apache.org/content/repositories/snapshots</url>
-      <releases>
-        <enabled>false</enabled>
-      </releases>
-      <snapshots>
-        <enabled>true</enabled>
-      </snapshots>
-    </pluginRepository>
-  </pluginRepositories>
```

Comment:
@nacx: I'm assuming we're basing this change on the idea that users have configured the Apache snapshots repo in their settings, as described in http://jclouds.apache.org/start/install/?